packages/mgt-spfx/config/package-solution.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/spfx-build/package-solution.schema.json",
+  "solution": {
+    "name": "mgt-spfx-client-side-solution",
+    "id": "e342d2ab-fc3f-45f6-93a5-268fd8632211",
+    "version": "1.0.0.0",
+    "includeClientSideAssets": true,
+    "skipFeatureDeployment": true,
+    "isDomainIsolated": false,
+    "developer": {
```

Comment:
Should these properties be updated with our details?

Reply:
Yes, good catch.

Reply:
We use https://privacy.microsoft.com/privacystatement for mgt.dev, we could probably use the same one here

Reply:
Updated